src/Algebra/Morphism/Construct/Composition.agda

```diff
@@ -423,3 +424,168 @@ module _ {K₁ : RawKleeneAlgebra a ℓ₁}
     ; surjective                  = Func.surjective (_≈_ K₁) (_≈_ K₂) (_≈_ K₃) F.surjective G.surjective
     } where module F = IsKleeneAlgebraIsomorphism f-iso; module G = IsKleeneAlgebraIsomorphism g-iso
 
+-----------------------------------------------------------------------
+-- Bundled morphisms between algebras
+------------------------------------------------------------------------
+
+-- Magma
+
+module _ {M₁ : Magma a ℓ₁}
```

Comment:
I think I'd avoid these anonymous modules in the `.Construct.*` modules, they're only ever going to have a single thing in them

Reply:
DONE.